server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -930,6 +959,23 @@ public boolean performConsumerUpdates(Consumer updated, Consumer toUpdate,
             }
         }
 
+        if (updated.getContentAccessMode() != null &&
+            !updated.getContentAccessMode().equals(toUpdate.getContentAccessMode()) &&
+            toUpdate.getType().isManifest()) {
+            if (!StringUtils.isBlank(updated.getContentAccessMode()) &&
```

Comment:
Its not exactly the same. in one case it checks the existing in one part and the incoming in the other.